### DISCUSSION TITLE
No ding events in wired doorbells

### DISCUSSION THREAD

**Discussion by pepepere** [tsightler/ring-mqtt#629] [General Discussion]
No ding event in all my wired doorbells with different HA installations... motion event, snapshots and others are working.
Has the API changed?
(my installations are with docker)

**Comment by tsightler** [comment]
As was already replied in the other thread, see https://github.com/dgreif/ring/issues/1212

Please [follow the steps there](https://github.com/dgreif/ring/issues/1212#issuecomment-1545783841) and report back.

**Reply by pepepere** [reply]
Hi, thx for your support!
I have a wired doorbell and the ding event don't work.
I've deleted all the devices and added only the "ring-mqtt-addon" and "missing device" (my installation is with docker), it has worked for 24 hours, then again it stopped notifying the ding event. I've deleted one more time all the devices, created a new token, and it's working again.
Any ideas?

**Reply by tsightler** [reply]
No ideas.  My current thought is that I'm just going to remove camera support from ring-mqtt entirely.  It's not worth the trouble at this point, IMO.

**Reply by pepepere** [reply]
More information... The integration in HA has the same problem, after a while the ding event stops working (here the only solution is to re-authenticate in the app)
It seems as if the token will expire in a short time, just for the ding event. There is no problem in the app, it always works fine. Could it be something that Ring does intentionally to increase subscriptions?

**Reply by Lucky-Luca234** [reply]
Same problem here. I am using the home assistant add-on and i recently got this problem for the first time. But since then i need to re-authenticate almost daily.
